util/opus.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$id": "https://kulturkrock.se/opus.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Opus",
+    "description": "An Opus to use for Screencrash",
+    "type": "object",
+    "properties": {
+        "startNode": {
+            "type": "string"
+        },
+        "nodes": {
+            "type": "object",
+            "patternProperties": {
+                "^[0-9]+.*$": {
```

Comment:
Technically, IDs don't need to start with numbers if the node has defined pdfPage and pdfLocationOnPage. I'm sure this is possible to encode with JSON schema somehow, or it could just be in the description if that is too complex.

Reply:
Leaving this for the future,